.travis.yml

```diff
@@ -3,12 +3,18 @@ jdk:
   - oraclejdk8
   - openjdk7
 
```

Comment:
He was referring to the lines below.  It should be a separate PR really as it's unrelated to this PR